lib/fastlane/plugin/wpmreleasetoolkit/versioning/formatters/build_code_formatter_derived_option_one.rb

```diff
@@ -0,0 +1,31 @@
+module Fastlane
+  module Wpmreleasetoolkit
+    module Versioning
+      class DerivedBuildCodeFormatter
+        # Calculate the next derived build code.
+        #
+        # This method derives a new build code from the given AppVersion object by concatenating the digit 1,
+        # the major version, the minor version, the patch version, and the build number.
+        #
+        # @param version [AppVersion] The AppVersion object to derive the next build code from.
```

Comment:
The YARD doc-comment expects the format for parameters to be `@param [Type] name The Description`. (Same feedback applies for all the documentation comments below that have the same format issue, of course)

```suggestion
        # @param [AppVersion] version The AppVersion object to derive the next build code from.
```

Reply:
Fixed in https://github.com/wordpress-mobile/release-toolkit/commit/7878e7c15dd78da1495dc2c7cabf804f8fb71049